be/src/io/cache/block_file_cache_factory.h

```diff
@@ -68,6 +70,14 @@ class FileCacheFactory {
      */
     std::string clear_file_caches(bool sync);
 
+    /**
+     * Clears data of the file you special.
```

Comment:
typo, it should be: Clear cache data of a specific segment file
plus, you may want to add another function that uses std::string segment_path as the second argument.